src/tests/functional/inference_engine/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -1090,3 +1091,158 @@ TEST_F(TransformationTestsF, eliminate_eltwise_dequantization_subgraph) {
     comparator.enable(FunctionsComparator::CmpValues::CONST_VALUES);
     comparator.enable(FunctionsComparator::CmpValues::ACCURACY);
 }
+
+enum class SplitType {
+    Split,
+    VariadicSplit
+};
+
+enum class RNNType : size_t {
```

Comment:
Why do we need `size_t` here?

Reply:
I've used values of RNNType enum as coefficients for input shape calculation
![image](https://user-images.githubusercontent.com/21181928/195327274-3c084dc1-1a66-455e-ba53-0e152725b065.png)
![image](https://user-images.githubusercontent.com/21181928/195327485-efcffb07-83a7-4438-a2dd-638c4a1f4e78.png)